docs/readmes/orc8r/dev_build.md

```diff
@@ -64,11 +64,18 @@ git checkout -b v1.6 origin/v1.6  # or desired Git target, e.g. master
 ```
 
 Build and publish Orchestrator images
+(Optionally, build and publish fluentd images for the Orchestrator by uncommenting the line `docker-compose.logging.yaml` in `build.py`.)
 
 ```bash
 cd ${MAGMA_ROOT}/orc8r/cloud/docker
 ./build.py --all
-for image in controller nginx ; do ${PUBLISH} -r ${REGISTRY} -i ${image} -v ${MAGMA_TAG} ; done
+for image in controller nginx ; do ${PUBLISH} -r ${REGISTRY} -i ${image} -v ${MAGMA_TAG} -u '' -p '' ; done
```

Comment:
What does the additional `-u '' -p ''` do?

Reply:
It sets the user and password. In my experience it is necessary, otherwise the command just gives this output but doesn't actually do anything else:
![image](https://user-images.githubusercontent.com/104822369/172960186-4f4a085a-12d9-48af-9261-e575279d940f.png)